source/Missions/Messages/NetworkMissionJoinInfo.cs

```diff
@@ -17,7 +23,7 @@ public class NetworkMissionJoinInfo : INetworkEvent
         [ProtoMember(2)]
         public readonly Vec3 StartingPosition;
         [ProtoMember(3)]
-        public readonly CharacterObject CharacterObject;
+        public CharacterObject CharacterObject { get; }
```

Comment:
If readonly works here, change everything in this class to use `readonly` for consistency